packages/block-editor/src/components/convert-to-group-buttons/toolbar.js

```diff
@@ -97,6 +102,13 @@ function BlockGroupToolbar() {
 					onClick={ onConvertToStack }
 				/>
 			) }
+			{ canInsertGrid && (
+				<ToolbarButton
+					icon={ grid }
+					label={ _x( 'Grid', 'verb' ) }
```

Comment:
This was a TIL for me, I didn't realise that grid could be used as a verb: https://en.wiktionary.org/wiki/grid#Verb. Nice!

Reply:
I wondered if the use here is clear enough? But the alternative would be a verbose or potentially confusing description like "configurable rows and columns". I guess ultimately what matters is if the meaning is sufficiently clear to translators; grid as a verb is unlikely to exist in every language.

Reply:
I think it's probably fine for now, particularly since it'd be an easy thing to update. I suppose another option would be to use `__( 'Grid' )` with no explanatory comment, to default to a noun 🤔